src/Components/Components/test/RendererTest.cs

```diff
@@ -3936,7 +3971,7 @@ private void Render()
                 => _renderHandle.Render(builder =>
                 {
                     builder.OpenElement(0, "my button");
-                    builder.AddAttribute(1, "my click handler", new Action<EventArgs>(eventArgs => OnClick(eventArgs)));
+                    builder.AddAttribute(1, "onmycustomevent", EventCallback.Factory.Create(this, eventArgs => OnClick(eventArgs)));
```

Comment:
The unit test that uses this class was falsely passing (as in, it would pass whether or not the behavior it was trying to test was working) ever since the move to `EventCallback`. It's hard to discover false positives like this, but I temporarily removed some of the logic to discover which unit test was covering it, and it turned out that none of them were.